conf/broker.conf

```diff
@@ -102,6 +102,12 @@ tlsAllowInsecureConnection=false
 # Using a value of 0, is disabling unackeMessage limit check and consumer can receive messages without any restriction 
 maxUnackedMessagesPerConsumer=50000
 
+# Max number of unacknowledged messages allowed per shared subscription. Broker will stop dispatching messages to
+# all consumers once, this limit reaches until consumer starts acknowledging messages back and unack count reaches
+# to maxUnackedMessagesPerDispatcher/2. Using a value of 0, is disabling unackedMessage-limit check and dispatcher
+# can dispatch messages without any restriction
+maxUnackedMessagesPerDispatcher=200000
```

Comment:
> . For example in the same test if we have 600 topics and each subscription allows 0.2million msgs then broker ends up with 120mln unack msgs in memory which doesn't help and for that this number should be lower. So, it's better to give option to configure this number.

Correct. Then would it be feasible to have an efficient way (maybe with relaxed enforcement) to add a per-broker limit?